walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -25,22 +25,54 @@
 public final class FindMeetingQuery {
```

Comment:
It seems all methods in this class could be static?

Reply:
That's true, the class state is not being used at all. Refactoring it now.